src/Mgate/SuiviBundle/Controller/MissionsController.php

```diff
@@ -42,36 +42,34 @@ public function modifierAction(Request $request, Etude $etude)
         }
 
         /* Form handling */
-        $form = $this->createForm(MissionsType::class, $etude, array('etude' => $etude));
+        $form = $this->createForm(MissionsType::class, $etude, ['etude' => $etude]);
         if ($request->getMethod() == 'POST') {
             $form->handleRequest($request);
 
             if ($form->isValid()) {
-
-                foreach ($form->get('missions') as $missionForm){
+                foreach ($form->get('missions') as $missionForm) {
                     $m = $missionForm->getData();
-                    foreach ($missionForm->get('repartitionsJEH') as $repartitionForm){
+                    foreach ($missionForm->get('repartitionsJEH') as $repartitionForm) {
                         $r = $repartitionForm->getData();
-                        /** @var RepartitionJEH $r */
+                        /* @var RepartitionJEH $r */
```

Comment:
this change looks wrong:
- IIRC an IDE won't pick up the typehint anymore if you replace the PHPDoc by a regular comment
- in-code comments should be done with `//` instead of `/*` (more subjective though...)